packages/react-card/src/components/CardFooter/__snapshots__/CardFooter.test.tsx.snap

```diff
@@ -6,11 +6,11 @@ exports[`CardFooter renders a default state 1`] = `
     class=""
   >
     Default CardFooter
-    <span
+    <div
```

Comment:
Intended change, action slot can contain any kind of elements so it should not be a `span`